ding_unilogin/js/ding_unilogin.js

```diff
@@ -3,7 +3,7 @@
  * JS to activate the UNI•Login button.
  */
 
-(function ($) {
+(function ($, Drupal) {
```

Comment:
Drupal is a global variable, not need to inject it.

```
(function ($) {
  Drupal.behaviors.exampleModule = {
    attach: function (context, settings) {
      // Code to be run on page load, and
      // on ajax load added here
    }
  };
}(jQuery));
```
This is what core JS do....

Reply:
Fixed in https://github.com/eReolen/ereolengo/pull/105/commits/d2f4244cb11710bf4833eb340732022cf0b1d976.